R/exclude.R

```diff
@@ -55,6 +55,17 @@ is_excluded_file <- function(file_exclusion) {
   Inf %in% file_exclusion[[names2(file_exclusion) == ""]]
 }
 
+line_info <- function(line_numbers) {
+  n <- length(line_numbers)
+  if (identical(n, 0L)) {
+    "no lines"
+  } else if (identical(n, 1L)) {
+    paste("line", line_numbers)
+  } else {
+    paste("lines", paste(line_numbers, collapse = ", "))
```

Comment:
I prefer toString() over paste(., collapse=', ')

Reply:
thx - first time I see that actually!